junit-platform-engine/src/main/java/org/junit/platform/engine/support/hierarchical/DefaultParallelExecutionConfigurationStrategy.java

```diff
@@ -133,7 +134,8 @@ public ParallelExecutionConfiguration createConfiguration(ConfigurationParameter
 	public static final String CONFIG_CUSTOM_CLASS_PROPERTY_NAME = "custom.class";
 
 	static ParallelExecutionConfigurationStrategy getStrategy(ConfigurationParameters configurationParameters) {
-		return valueOf(configurationParameters.get(CONFIG_STRATEGY_PROPERTY_NAME).orElse("dynamic").toUpperCase(Locale.ENGLISH));
+		return valueOf(
+			configurationParameters.get(CONFIG_STRATEGY_PROPERTY_NAME).orElse("dynamic").toUpperCase(Locale.ENGLISH));
```

Comment:
Shouldn't it be `Locale.ROOT` instead of `Locale.ENGLISH`?

> Useful constant for the root locale. The root locale is the locale whose language, country, and variant are empty ("") strings. This is regarded as the base locale of all locales, and is used as the language/country neutral locale for the locale sensitive operations.

https://docs.oracle.com/javase/8/docs/api/java/util/Locale.html#ROOT

Reply:
I did not know the existence of Locale.ROOT and I was sure that uppercase of **i** is **I** in English Locale.  That can be changed to ROOT, dont know which is the best choice.

Reply:
`Locale.ROOT` is used internally in Kotlin's equivalent [`uppercase()`](https://github.com/JetBrains/kotlin/blob/769f4c630f7b3ab326cd32f643cb1def6663f8c5/libraries/stdlib/jvm/src/kotlin/text/StringsJVM.kt#L147) function, so that's what I'd be tempted to use.